core/src/test/scala/com/rackspace/com/papi/components/checker/wadl/WADLCheckerHeaderSingleSpec.scala

```diff
@@ -146,7 +146,7 @@ class WADLCheckerHeaderSingleSpec extends BaseCheckerSpec with LogAssertions {
                XML.loadFile("src/test/resources/xsd/test-urlxsd.xsd"))
       When("the wadl is translated")
       val checker = builder.build (inWADL, TestConfig(false, false, true, true, true, 1,
-                                                      true, true, true, "XalanC",
+                                                      true, false, true, "XalanC",
```

Comment:
These "checker" tests -- essentially confirm that the sate machine is created as expected. None of these tests test functionality related to the Grammar Transform, that's not the purpose of these tests, but in the past checking grammar didn't really affect the state machine in a way that the assertions in these tests check. Now that warnings are enabled by default the state machines changed so to prevent further changes, I've simply disabled setting the grammar transform in the config here.  Actual grammar transform tests are checked elsewhere.